### PR TITLE
🐌 SNPE env and example fix

### PR DESCRIPTION
**Pull request by trajepl**
## Describe your changes

There are a few issues for current SNPE examples:
1. The libxx.so under customized `python3.6` need to be put visible for system for SNPE cmd running
    - put `str(Path(f"{snpe_root}/python36-env/lib"))` under `LD_LIBRARY_PATH` can fix.
2. SNPE evaluation extent the logits wrongly, 
    - fixed in this PR.
3. We did not support tensorflow model evaluation.
    - update examples without input model evaluation.
4. Wrong metrics config for inception examples
5. Importing issue for vgg, tell that in README.md to let user update it manually as a workaround.
6. Python3.6 seems a bit outdated for latest SNPE updates. The vgg example failed to be run. Updated the python env to 3.8 can fix.

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Lint and apply fixes to your code by running `lintrunner -a`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

**Review by jambayk — APPROVED** [review]
Thanks for the fixes. The lib and bin paths worked fine previously so I am not sure if it's something to do with the new snpe version or our environment differences. But it doesn't hurt to add extra paths.

**Comment by guotuofeng** [comment]
It seems the shellcheck fail, do we need fix it?

**Review by guotuofeng — CHANGES_REQUESTED** [review]
do we need fix the shellcheck failure?

**Comment by trajepl** [comment]
> do we need fix the shellcheck failure?

Thanks! Added the Shebang info for shell script.

**Review by guotuofeng — APPROVED** [review]
